### DISCUSSION TITLE
Docker minecraft start server script

### DISCUSSION THREAD

**Discussion by florensm** [itzg/docker-minecraft-server#1868] [Q&A]
Hi, 

Is there a way I can change the minecraft start-up script? I used to use those myself but i'm new to docker. Now my server is 1.19.3 vanilla but i want to use 1.19.2 Paper, so i downloaded the paper jar with curl, now its in that directory but how do i change the startup script so it takes another name?

**Comment by itzg** [comment]
You're missing out on the major features of this image 😀

From [here](https://github.com/itzg/docker-minecraft-server#running-a-paper-server), you can see you just need to set `TYPE` to "PAPER" and `VERSION` to "1.19.2" and it'll download and run that one for you.

**Comment by florensm** [comment]
I did use that but like if i want to use a custom spigot, how can i do that?

**Reply by itzg** [reply]
You said Paper...I'm confused. 

Maybe this would help https://github.com/itzg/docker-minecraft-server#running-with-a-custom-server-jar

or https://github.com/itzg/docker-minecraft-server#running-a-bukkitspigot-server

**Comment by florensm** [comment]
And where can i use that: CUSTOM_SERVER? I just used docker run. Where can i find all those values?

**Comment by itzg** [comment]
You use that as environment variables passed to the container, such as `-e CUSTOMER_SERVER=server.jar`

Everything is documented [in the README](https://github.com/itzg/docker-minecraft-server#readme) I have been linked you to.
